docs/securing_fledge.rst

```diff
@@ -56,8 +66,11 @@ The default installation of a Fledge service comes with security features turned
 
   - Fledge maintains full audit logs of all updates to the Fledge configuration, and other events. This allows for complete auditing of who made what changes to the Fledge configuration and when the changes were made.
 
+  - Fledge can have optional allow and deny lists configured that list those hosts that are either permitted to connect to the UI and REST API or are denied from access.
+
 .. note::
-  Although all these features add security to a Fledge installation it is still recommended to run Fledge within firewalls and limit those networks that have access to the Fledge API port when using for production purposes.
+
+   Although all these features add security to a Fledge installation it is still recommended to run Fledge within firewalls and limit those networks that have access to the Fledge API port when using for production purposes.
```

Comment:
it is still recommended to run Fledge behind firewalls and restrict access to the Fledge API port to trusted networks when using it in production environments.